### PR TITLE
Feature/COR-1341-disable-7-day-timeframe

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard21**
## Summary

* removed TimeframeOption 7 days from TimeframeOptionList
* refactored sewer chart
* refactored archived graph

#### Before
<details>
<summary>Toggle before screenshots</summary>

before_timeselection_graph
before_archived_graph

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

after_timeselection_graph
after_sewer_chart
after_archived_graph

</details>

### CODE REVIEW

File: packages/common/src/utils/timeframe/index.ts [hunk 1]
```diff
@@ -18,14 +11,7 @@ export enum TimeframeOption {
...
 
-export const TimeframeOptionsList = [
-  TimeframeOption.ALL,
-  TimeframeOption.ONE_WEEK,
-  TimeframeOption.THIRTY_DAYS,
-  TimeframeOption.THREE_MONTHS,
-  TimeframeOption.SIX_MONTHS,
-  TimeframeOption.LAST_YEAR,
-];
+export const TimeframeOptionsList = [TimeframeOption.ALL, TimeframeOption.THIRTY_DAYS, TimeframeOption.THREE_MONTHS, TimeframeOption.SIX_MONTHS, TimeframeOption.LAST_YEAR];
```
Comment (VWSCoronaDashboard18) [inline packages/common/src/utils/timeframe/index.ts:14]: I thought we opted for still having the ONE_WEEK available. But excluded of using it...

Comment (VWSCoronaDashboard26) [reply packages/common/src/utils/timeframe/index.ts:14]: Please correct me if I am wrong, but is that not exactly what is achieved here? We still have the original `enum` (containing both `ONE_WEEK` and `FIVE_WEEKS`) on lines 4 through 12, yet we simply exclude it from the `TimeframeOptionsList` array?

Comment (VWSCoronaDashboard18) [reply packages/common/src/utils/timeframe/index.ts:14]: As you showed it is still in the ENUM! Great job!

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Looks great

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Pull request merged**